draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -102,12 +102,11 @@ has to verify that all disclosed claim values were part of the original
 Issuer-signed JWT. The Verifier will not, however, learn any claim
 values not disclosed in the Disclosures.
 
-This document also specifies an optional mechanism for Key Binding,
-which is the concept of binding an SD-JWT to a Holder's public key
-and requiring that the Holder prove possession of the corresponding
-private key when presenting the SD-JWT.
-The strength of the binding is conditional upon the trust
-in the protection of the private key of the key pair an SD-JWT is bound to.
+For SD-JWTs where the issuer has specified an associated public key for the
+Holder (e.g., using the `cnf` claim [@?RFC7800]), this document also defines a
+format for SD-JWTs with Key Binding (SD-JWT-KB).  By sending an SD-JWT-KB to a
```

Comment:
```suggestion
format for SD-JWTs with Key Binding (SD-JWT-KB).  By optionally sending an SD-JWT-KB to a
```
I think the nuance of `specifies an _optional_ mechanism for key binding` of the original text should be reflected better